### ISSUE TITLE
Travis fails

### ISSUE THREAD

**Issue by smolkaj** [frenetic-lang/frenetic#516]
Travis builds currently fail (only on the master branch) when trying to build/update (?) the documentation:
```
...
Updating docs on Github pages...
.travis-docgen.sh: line 21: KEEP: unbound variable
The command "bash -ex .travis-docgen.sh" exited with 1.
```

I figured out where the problem occurs: https://github.com/ocaml/ocaml-ci-scripts/blob/master/.travis-docgen.sh#L21
But I don't know how to fix it. In particular, where would one have to set the KEEP variable? @craig-riecke, do you know what to do?

**Comment by smolkaj** [comment]
Maybe the problem is not on on our end, see https://github.com/ocaml/ocaml-ci-scripts/issues/121#issuecomment-232912452.

**Comment by craig-riecke** [comment]
Fixed by #518 .
